src/components/redux/State.js

```diff
@@ -30,13 +29,28 @@ const state = {
     }
 };
 
-const addPost = (postMessage) => {
+const addPost = () => {
     let newPost = {
       id: 5,
-      message: postMessage , 
+      message: state.ProfilePage.newPostText , 
       likesCount: 0
     };
     state.ProfilePage.postsData.push(newPost);
+    state.ProfilePage.newPostText = ('');
     rerenderEntireTree(state);
   };
-export {state,addPost};
\ No newline at end of file
+
+let rerenderEntireTree = () => {
+
+}
+
+  const updateNewpostText = (newText) => {
+    state.ProfilePage.newPostText = newText;
+    rerenderEntireTree(state);
+  }
+
+  const subscribe = (observer) => {
+    rerenderEntireTree = observer;
+
+  }
```

Comment:
Consider adding an empty line after the last "}".

Reply:
done